Flick/Cells/ProfileSummaryTableViewCell.swift

```diff
@@ -118,17 +98,18 @@ class ProfileSummaryTableViewCell: UITableViewCell {
         }
 
         friendsPreviewView.snp.makeConstraints { make in
-            make.leading.equalTo(usernameLabel.snp.trailing).offset(padding)
+            make.leading.equalTo(usernameLabel.snp.trailing)
             make.top.bottom.height.equalToSuperview()
-            make.width.equalTo(friendsPreviewWidth)
+            make.width.equalTo(0)
```

Comment:
why width = 0?

Reply:
i made it 0 cuz initially there are no friends so technically the friendsPreviewView should not be there. But in configure() we passed in friends so we can just update width instead of remaking the width constraint